### PR TITLE
COMPASS-213: Pass readonly options to collections

### PR DESCRIPTION
**Pull request by durran**

### CODE REVIEW

File: lib/fetch.js [hunk 1]
```diff
@@ -338,7 +338,8 @@ function parseCollection(resp) {
   return {
     _id: ns.toString(),
     name: ns.collection,
-    database: ns.database
+    database: ns.database,
+    readonly: resp.info ? resp.info.readOnly : false
```
Comment (imlucas) [inline lib/fetch.js]: `_.get(resp, 'info.readOnly', false)`

File: test/fetch-mocked.test.js [hunk 2]
```diff
@@ -267,22 +271,26 @@ describe('fetch-mocked', function() {
           {
             '_id': 'accounts.testCol',
             'database': 'accounts',
-            'name': 'testCol'
+            'name': 'testCol',
+            'readOnly': true
```
Comment (imlucas) [inline test/fetch-mocked.test.js]: Shouldn't this be `'readonly': true`?

**Review by imlucas — CHANGES_REQUESTED** [review]

**Review by durran — COMMENTED** [review]
Changes made and collection model bumped.

**Comment by aherlihy** [comment]
LGTM

**Pull request merged**